### PR TITLE
IT-3712 | deploy dist on master

### PR DESCRIPTION
**Pull request by przemyslawjanpietrzak**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+          node-version: '14.15'
+      - name: Install and Build 🔧
+        run: | # Install npm packages and build the Storybook files
+          yarn install --pure-lockfile
+          yarn build
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@4.1.1
+        with:
+          branch: master # The branch the action should deploy to.
+          folder: dist # The folder that the build-storybook script generates files.
```
Comment (kvas-damian) [inline .github/workflows/build.yml:25]: stroybook?

Comment (przemyslawjanpietrzak) [reply .github/workflows/build.yml:25]: copy paste, my bad

Comment (przemyslawjanpietrzak) [inline .github/workflows/build.yml:25]:
```suggestion
          folder: dist # The folder that the build script generates files.
```

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+          clean: false # Automatically remove deleted files from the deploy branch
+          target-folder: dist # The folder that we serve our Storybook files from
```
Comment (przemyslawjanpietrzak) [inline .github/workflows/build.yml:27]:
```suggestion
          target-folder: dist # The folder that we serve our design-system files from
```

**Pull request merged**